### DISCUSSION TITLE
How do i not log internal IP's?

### DISCUSSION THREAD

**Discussion by ToontjeM** [telekom-security/tpotce#1097] [Q&A]
I have my t-pot on an isolated network with one other host. I use that host to scan the t-pot for vulnerabilities using Greenbone. How can i not log those scans in t-pot so my logs stay clean from internal IP addresses? Greenbone creates quite a bit of noise when scanning and t-pot catches everything.

**Comment by t3chn0m4g3** [comment]
There is no option in T-Pot to exclude PIRs.

**Reply by Ztaguassi** [reply]
We are using a modified copy of blackhole script to add the IPs to blackhole so it will not show in T-Pot
